packages/terra-folder-tree/src/FolderTree.jsx

```diff
@@ -1,26 +1,50 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { injectIntl } from 'react-intl';
-// import classNames from 'classnames/bind';
-// import styles from './FolderTree.module.scss';
+import classNames from 'classnames/bind';
 
-// const cx = classNames.bind(styles);
+import ActionHeader from 'terra-action-header';
+
+import FolderTreeItem from './subcomponents/FolderTreeItem';
+import styles from './FolderTree.module.scss';
+
+const cx = classNames.bind(styles);
 
 const propTypes = {
   /**
-   * @private
-   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
+   * List of FolderTree.Items to be displayed as content within the FolderTree.
+   */
+  children: PropTypes.node.isRequired,
```

Comment:
Will there be any issues to making this required? What if a consumer does `<FolderTree><FolderTree/>` and dynamically adds the children later?

Reply:
My thought in marking this as required was that without any FolderTreeItems, it's really not a folder tree at all but a floating header. But it doesn't really hurt to make this optional since it's up to consumers what content to put in the tree.